### DISCUSSION TITLE
请问如何添加本地大模型。以及我在‘添加LocalAI'时报错

### DISCUSSION THREAD

**Discussion by tanpengxiong** [langgenius/dify#3967] [Help]
一、我本地安装了一个chatglm3的大模型，如何在dify添加本地大模型
二、我在添加LocalAI时报错
http://192.168.30.222/console/api/workspaces/current/model-providers/localai/models
报错  400 BAD REQUEST
![5e583dc4ebd472bda93a4a2d6d3beea](https://github.com/langgenius/dify/assets/14973426/d3b4d848-d299-436b-a36b-d015b0e87666)
![92f66887d35c4429940c48d776b03cd](https://github.com/langgenius/dify/assets/14973426/241424dc-1ecb-4f83-bd8f-3a9e044a70bd)

**Comment by crazywoola** [comment]
Please make sure following our language policy #1542
